js/poloniexfutures.js

```diff
@@ -0,0 +1,1628 @@
+                'remaining': undefined,
+                'status': undefined,
+                'fee': undefined,
+                'trades': undefined,
+                'timeInForce': undefined,
+                'postOnly': undefined,
+                'stopPrice': undefined,
+                'info': response,
+            });
+        }
+        return result;
+    }
+
+    async fetchOrdersByStatus (status, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name poloniexfutures#fetchOrdersByStatus
+         * @description fetches a list of orders placed on the exchange
+         * @see https://futures-docs.poloniex.com/#get-order-list
+         * @see https://futures-docs.poloniex.com/#get-untriggered-stop-order-list
+         * @param {string} status 'active' or 'closed', only 'active' is valid for stop orders
+         * @param {string|undefined} symbol unified symbol for the market to retrieve orders from
+         * @param {int|undefined} since timestamp in ms of the earliest order to retrieve
+         * @param {int|undefined} limit The maximum number of orders to retrieve
+         * @param {object} params exchange specific parameters
+         * @param {bool|undefined} params.stop set to true to retrieve untriggered stop orders
+         * @param {int|undefined} params.until End time in ms
+         * @param {string|undefined} params.side buy or sell
+         * @param {string|undefined} params.type limit or market
+         * @returns An [array of order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const stop = this.safeValue (params, 'stop');
+        const until = this.safeInteger2 (params, 'until', 'till');
+        params = this.omit (params, [ 'stop', 'until', 'till' ]);
+        if (status === 'closed') {
+            status = 'done';
+        } else if (status === 'open') {
+            status = 'active';
+        }
+        const request = {};
+        if (!stop) {
+            request['status'] = status;
+        } else if (status !== 'active') {
+            throw new BadRequest (this.id + ' fetchOrdersByStatus() can only fetch untriggered stop orders');
+        }
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['symbol'] = market['id'];
+        }
+        if (since !== undefined) {
+            request['startAt'] = since;
+        }
+        if (until !== undefined) {
+            request['endAt'] = until;
+        }
+        const method = stop ? 'privateGetStopOrders' : 'privateGetOrders';
+        const response = await this[method] (this.extend (request, params));
+        const responseData = this.safeValue (response, 'data', {});
+        const orders = this.safeValue (responseData, 'items', []);
+        return this.parseOrders (orders, market, since, limit);
```

Comment:
ccxt distinguishes closed orders from canceled orders, whereas poloniex don't, so we should probably filter them out here, no?
![image](https://user-images.githubusercontent.com/43336371/210087696-bb208a73-dbc1-4af9-a48c-80cb914e021f.png)